src/features/genericContentPage/renderMethods.js

```diff
@@ -45,9 +44,9 @@ const renderTypeformButton = ({ primary, content, to }) => {
         }}
         autoClose={5}
       />
-      <StyledButton primary={primary} onClick={openForm}>
+      <Button variant={primary ? 'primary' : 'secondary'} onClick={openForm}>
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Prop `onClick` must be placed on a new line](https://app.codacy.com/manual/PrideInLondon/pride-london-web/pullRequest?prid=4765408)